### DISCUSSION TITLE
Is MinIO needed for self-hosted Ente-Auth ?

### DISCUSSION THREAD

**Discussion by jpbaril** [ente-io/ente#3145] [Q&A] [answered]
I followed this guide: https://github.com/ente-io/ente/blob/main/server/docs/docker.md
I'm wondering, if I intend to only use Ente Auth app to manage MFA codes, do I need to deploy MinIO service ?

**Comment by mnvr** [comment, marked as answer]
Right, it's not needed. You currently don't need to configure the S3 object storage (e.g. minio containers) if you're only using your self hosted Ente instance for auth.

**Comment by timkgh** [comment]
Any plans to offer a light version of Ente Auth that can use SQLite for example instead of a heavier database? For home use, 2-3 users, a single lightweight container would be perfect, like Vaultwarden vs Bitwarden self-hosted.
